tests/pipeline/test_pipeline.py

```diff
@@ -235,6 +236,53 @@ def test_with_custom_config(self, snapshot: SnapshotTest):
 
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
 
+    def test_with_custom_config_with_absolute_defaults_path(
+        self, snapshot: SnapshotTest
+    ):
+        with open(
+            str(RESOURCE_PATH / "custom-config/config.yaml"), "r"
```

Comment:
casting to `str` is unnecessary, `open` supports `Path` instances :)

Reply:
Cool, removed the unnecessary step